doc/users/whats_new/updated_backend_wx.rst

```diff
@@ -0,0 +1,12 @@
+wx backend has been updated
+---------------------------
+The wx backend can now be used with both wxPython classic and
+`Phoenix <http://wxpython.org/Phoenix/docs/html/main.html>`__.
+
+wxPython classic has to be at least version 2.8.12 and works on Python 2.x,
+wxPython Phoenix needs a current snapshot and works on Python 2.7 and 3.4+.
```

Comment:
@WeatherGod I still think this needs some kind of version specification for WX Phoenix. I don't know if it is likely that there will be an official release before the next Matplotlib release. 

I would replace current with a date/month something like  "As of April 2015 no official wxPhoenix release has been made but a recent snapshot should work ..." Followed by the python versions supported.